Annotations/HideFreeTextBorder/App.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version = "1.0" encoding = "UTF-8" ?>
```

Comment:
<img width="267" alt="image" src="https://github.com/user-attachments/assets/7bd50a51-9ff3-4feb-aaec-48b5dfb3874f" />

Why the document name comes with suffix 1?